### ISSUE TITLE
Delete command and landings do not remove radar target

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#743]
Browser: Chrome
Airport: All

#### Issue Description
I don't know how it can still be messed up, but there are still problems removing radar targets when aircraft are removed. In this case, it's use of the `del` or `kill` command that does not remove the radar target. Also landing aircraft are not removed.

#### Steps to reproduce
1. To any controllable aircraft, use the `del` or `kill` command 
1. Observe that the target is still present, forever frozen in its last known position.

![image](https://user-images.githubusercontent.com/5103735/30952924-cf060b54-a3f7-11e7-857c-e073fc0b70d8.png)

**Comment by n8rzz** [comment]
@erikquinn you're going to laugh...

```javascript
aircraft_remove(aircraftModel) {
    AirportController.removeAircraftFromAllRunwayQueues(aircraftModel);
    this.removeFlightNumberFromList(aircraftModel);
    this.removeAircraftModelFromList(aircraftModel);
    this._removeTransponderCodeFromUse(aircraftModel);
    this.removeAllAircraftConflictsForAircraft(aircraftModel);
    this.removeStripView(aircraftModel);
    this._scopeModel.radarTargetCollection.removeRadarTargetModelForAircraftModel(aircraft);
}
```

the bug is in that block, can you see it?

**Comment by erikquinn** [comment]
Oh for gods sake... :man: :gun:

**Comment by erikquinn** [comment]
Resolved by #750. Closing.
